llvm/lib/Target/RISCV/RISCVSchedSiFive7.td

```diff
@@ -200,6 +200,7 @@ def SiFive7Model : SchedMachineModel {
                              HasStdExtZcmt, HasStdExtZknd, HasStdExtZkne,
                              HasStdExtZknh, HasStdExtZksed, HasStdExtZksh,
                              HasStdExtZkr];
+  let MacroFusions = [LUIADDI];
```

Comment:
SiFive7 doesn't implement any macrofusions. Is there some other way we can test?

Reply:
Good question! I was thinking about testability too since we can't provide subtarget feature in `-mattr` anymore. Do you have any suggestion?

Reply:
I posted #73075 for enhancing testability.